server/manager/src/main/java/org/apache/accumulo/manager/ManagerClientServiceHandler.java

```diff
@@ -623,34 +612,43 @@ public void requestTabletHosting(TInfo tinfo, TCredentials credentials, String t
     manager.mustBeOnline(tableId);
 
     final List<KeyExtent> success = new ArrayList<>();
-    final Ample ample = manager.getContext().getAmple();
-    try (var mutator = ample.conditionallyMutateTablets()) {
-      extents.forEach(e -> {
-        log.info("Tablet hosting requested for: {} ", KeyExtent.fromThrift(e));
-        KeyExtent ke = KeyExtent.fromThrift(e);
-        if (recentHostingRequest.getIfPresent(ke) == null) {
-          mutator.mutateTablet(ke).requireAbsentOperation()
-              .requireTabletAvailability(TabletAvailability.ONDEMAND).requireAbsentLocation()
-              .setHostingRequested().submit(TabletMetadata::getHostingRequested);
-        } else {
-          log.trace("Ignoring hosting request because it was recently requested {}", ke);
-        }
+    final List<KeyExtent> inProgress = new ArrayList<>();
+    extents.forEach(e -> {
+      log.info("Tablet hosting requested for: {} ", KeyExtent.fromThrift(e));
```

Comment:
Suggest moving this log statement down a few lines to where the KeyExtent is added to `inProgress` - basically when it's been added to `hostingRequestInProgress` and doesn't already exist.